applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -324,6 +325,8 @@ static void syncable_cb(struct bt_audio_broadcast_sink *sink, bool encrypted)
 	int ret;
 	static uint8_t bis_encryption_key[16];
 
+	LOG_DBG("Broadcast sink is syncable");
```

Comment:
Several of these changes look identical to the changes in https://github.com/nrfconnect/sdk-nrf/pull/10227

Reply:
Yes as I believe they are log messages and needed because both are button 5 processes